pkg/flink/internal/controller/input_controller.go

```diff
@@ -161,6 +161,10 @@ func (c *InputController) Prompt() prompt.IPrompt {
 			ASCIICode: []byte{0x1b, 0x66},
 			Fn:        prompt.GoRightWord,
 		}),
+		prompt.OptionAddASCIICodeBind(prompt.ASCIICodeBind{
+			ASCIICode: []byte{0x1b, 0x7F},
```

Comment:
Can you leave a comment explaining what 0x1b and 0x7F mean? It would be nice not to need to consult an ASCII chart in the future

Reply:
Yep, will add some explanation to all those bindings!